examples/bevy_state.rs

```diff
@@ -0,0 +1,100 @@
+use bevy::{
+    input::Input,
+    prelude::{App as BevyApp, AssetServer, Commands, KeyCode, Res, ResMut, State, SystemSet},
+    window::WindowDescriptor,
+    DefaultPlugins,
+};
+use kayak_ui::bevy::{BevyContext, BevyKayakUIPlugin, FontMapping, UICameraBundle};
+use kayak_ui::core::{render, Index};
+use kayak_ui::widgets::{App, Text};
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+enum GameState {
+    MainMenu,
+    Options,
+    Play,
+}
+
+fn create_main_menu(mut commands: Commands) {
+    let context = BevyContext::new(|context| {
+        render! {
+            <App>
+                <Text content={"Main Menu".to_string()} size={32.0} />
```

Comment:
Might be worth putting instructions to press space to switch to the next state. Just to flesh it out a bit more.